x-pack/plugins/security_solution/public/common/utils/timeline/use_show_timeline.tsx

```diff
@@ -6,29 +6,43 @@
  */
 
 import { useState, useEffect } from 'react';
+import { matchPath } from 'react-router-dom';
 import { useRouteSpy } from '../route/use_route_spy';
 
-const hideTimelineForRoutes = [`/cases/configure`, '/administration', 'rules/create'];
+import { getLinksWithHiddenTimeline } from '../../links';
+
+const DEPRECATED_HIDDEN_TIMELINE_ROUTES: readonly string[] = [
+  `/cases/configure`,
+  '/administration',
+  '/rules/create',
+  '/get_started',
+  '/threat_hunting',
+  '/dashboards',
+  '/manage',
+];
+
+const isTimelineHidden = (currentPath: string, isGroupedNavigationEnabled: boolean): boolean => {
+  const groupLinksWithHiddenTimelinePaths = getLinksWithHiddenTimeline().map((l) => l.path);
+
+  const hiddenTimelineRoutes = isGroupedNavigationEnabled
+    ? groupLinksWithHiddenTimelinePaths
+    : DEPRECATED_HIDDEN_TIMELINE_ROUTES;
+
+  return !!hiddenTimelineRoutes.find((route) => matchPath(currentPath, route));
+};
 
 export const useShowTimeline = () => {
-  const [{ pageName, pathName }] = useRouteSpy();
+  // TODO use this hook: useIsGroupedNavigationEnabled();
+  const isGroupedNavigationEnabled = false;
```

Comment:
`useIsGroupedNavigationEnabled ` is on Sergi's branch. We need to update this when he merges it.